.buildkite/pipeline.yml

```diff
@@ -323,7 +322,6 @@ steps:
       - rake test:ios:build_xcode
 
   - label: ':ios: Generate test fixture Xcode project for Unity 2019'
-    skip: Pending PLAT-6447
```

Comment:
I've just noticed that the next step (that actually builds the fixture) is still skipped.  Given that we don't actually run the tests yet that's perhaps less of an issue - it depends on out appetite for incurring another CI run at this stage.